src/scss/components/_icon.scss

```diff
@@ -1,10 +1,14 @@
 .icon {
-  display: inline-block;
-  margin: 2px 20px 0;
+  display: flex;
   vertical-align: top;
+  font-style: italic;
+  line-height: 1;
+  color: get-color((color: blackNeutral, opacity: .4));
+  align-items: center;
 
   img {
```

Comment:
@iondrimba I think that is a great opportunity to change `img` to a class. Scorpion project has a svg inside `icon` block already. What do you think?